### ISSUE TITLE
FocalLoss multiclass alpha

### ISSUE THREAD

**Issue by lukazso** [qubvel/segmentation_models.pytorch#808]
Hey,

first of all thanks for the work with this repo. I am confused with the multiclass implementation of the focal loss. In my understanding, the parameter for the class weight factor `alpha` in a multiclass setting should not be a scalar, but a tensor of length `num_classes` to provide a class weight for each class separately. To me it seems that it does not make sense to provide a single scalar `alpha` for multiple classes, as each class will receive the same weight factor then. For example, the parameters of a function `focal_loss_multiclass(...)` should look like this:

```python
def focal_loss_multiclass(pred_logits: torch.Tensor, gt: torch.Tensor, alpha: torch.Tensor, gamma: float|torch.Tensor):
    """ Calculates focal loss for multiclass problems.

    Args:
        pred_logits (torch.Tensor): logits of shape [b, num_classes, dim], or for images [b, num_classes, h, w]
        gt (torch.Tensor): tensor of shape [b, dim], or for images [b, h, w] with ground truth class indices
        alpha (torch.Tensor): class weights of shape [num_classes]
        gamma (float | torch.Tensor): factor to balance easy vs. hard samples

    Returns:
        focal loss as a torch.Tensor
    """

    # do something here

```

In the current implementation it is not possible. Am I wrong here?
